capstone/src/main/webapp/youtubeScript.js

```diff
@@ -108,6 +112,12 @@ function getAttributeTotals(attributeScores) {
   const attributeTotals = new Map();    
   for (let i = 0; i < requestedAttributes.length; i++) {
     for (let j = 0; j < attributeScores.length; j++) {
+      // populates attributeData to support CSV output and attributeTotals to support averaging
+      if(attributeData[j] == null) {
```

Comment:
nit: Let's put a space after the `if` and in the comment above, `populates` -> `Populates`

Reply:
addressed in "address Daniel's and Chris's nits"